src/Raven.Client/Documents/Commands/Batches/DeleteCommandData.cs

```diff
@@ -9,24 +9,38 @@ namespace Raven.Client.Documents.Commands.Batches
     public class DeleteCommandData : ICommandData
     {
         public DeleteCommandData(string id, string changeVector)
+            :this(id, changeVector, changeVector)
+        {
+            
+        }
+        public DeleteCommandData(string id, string changeVector, string oldChangeVector)
         {
             Id = id ?? throw new ArgumentNullException(nameof(id));
             ChangeVector = changeVector;
+            OldChangeVector = oldChangeVector;
         }
 
+        public string OldChangeVector { get; }
         public string Id { get; }
         public string Name { get; } = null;
         public string ChangeVector { get; }
         public CommandType Type { get; } = CommandType.DELETE;
+        public BlittableJsonReaderObject Document { get; set; }
```

Comment:
Why is it called "Document" while we're assigning document metadata to it?

Reply:
No longer needed, removed.